tes-adapter/src/main/resources/application.properties

```diff
@@ -7,4 +7,6 @@ cloud.pipeline.hddSize=50
 cloud.pipeline.preemtible=true
 cloud.pipeline.region=eu-central-1
 cloud.pipeline.service.name=CloudPipeline
-cloud.pipeline.doc=https://epam.github.io/cloud-pipeline/
\ No newline at end of file
+cloud.pipeline.doc=https://epam.github.io/cloud-pipeline/
+security.allowed.client.ip.range=192.168.1.0/24
```

Comment:
also add this property to test application.properties in order to be more straight forward